README.md

```diff
@@ -1,2 +1,62 @@
 # inverted-index
 Inverted index data structure built with JavaScript
+
+[![Build Status](https://travis-ci.org/andela-korjiewuru/inverted-index.svg?branch=development)](https://travis-ci.org/andela-korjiewuru/inverted-index)
+[![Code Climate](https://codeclimate.com/github/andela-korjiewuru/inverted-index/badges/gpa.svg)](https://codeclimate.com/github/andela-korjiewuru/inverted-index)
+[![Coverage Status](https://coveralls.io/repos/github/andela-korjiewuru/inverted-index/badge.svg?branch=development)](https://coveralls.io/github/andela-korjiewuru/inverted-index?branch=development)
+
+### dexIt Inverted Index Application
+
+dexIt enables you to upload your JSON array files, create an index map of words in to the corresponding JSON document in the array, and perform search on the words present.
+
+#### Using the App
+
+When the app is installed, follow the guide below to use InvertedIndex App
+
+1. Click on `Uploadfile` to upload a file(s)
+2. Upon successful upload, the select bxz in 'File upload detials' section to choose from uploaded file(s).
+3. Click on `Get Index` to create the Index for the selected file
+4. View the created Indices displayed in a tabular format in the right corner under `Table of Indices`
+5. You can searh through one or all files that have already been indexed.
+
+###### Note:
+* You may want to use the `What is this?` and `Usage guide` in the application menu to get more acquinted with the app.
+* You can do multiple uploads of JSON files
+
+#### Prerequisites
+
+You should have the following technologies installed to use the app
+
+```
+node
+```
+
+#### Installing
+
+To clone the repo, run the following command
+
+```
+git clone https://github.com/andela-korjiewuru/inverted-index.git
+```
+
+#### Install the project dependencies
+
+```
+npm install
```

Comment:
Is `npm install` a project dependency? Dependencies are external libraries / frameworks that your application depends on, especially for local development.